radeon-profile/ioctl_amdgpu.cpp

```diff
@@ -77,8 +77,7 @@ bool amdgpuIoctlHandler::getCoreClock(int *data) const {
 #ifdef AMDGPU_INFO_SENSOR_GFX_SCLK
     return getSensorValue(data, sizeof(data), AMDGPU_INFO_SENSOR_GFX_SCLK);
 #else
-    Q_UNUSED(data);
-    return false;
+    return getClocks(data, NULL);
```

Comment:
VCE clocks are not the same as GPU core and memory clocks and can't be used as fallback.